snuba/datasets/schema.py

```diff
@@ -75,11 +70,59 @@ def get_column_differences(self, expected_columns: Mapping[str, str]) -> List[st
 
     def get_migration_statements(
         self
-    ) -> Callable[[str, Mapping[str, str]], Sequence[str]]:
+    ) -> Optional[Callable[[str, Mapping[str, str]], Sequence[str]]]:
         return self.__migration_function
 
 
 class TableSchema(Schema):
```

Comment:
This should become the TableStorage in the overall redesign since it represents one single table.

Reply:
How do you feel about calling it `ClickHouseSchema` or something more clickhouse specific? Cause these are going to be actual tables in clickhouse, if I understand correctly. It also lines up well with the way we have been talking about it - i.e. clickhouse schemas vs query schemas.

Reply:
I would not add it now, because , as of now, we are not making a distinction between the two schemas.